src/features/user/TreeMapper/Analytics/index.tsx

```diff
@@ -1,37 +1,83 @@
 import type { Project } from '../../../common/Layout/AnalyticsContext';
-import type { APIError } from '@planet-sdk/common';
-import type { MapProject } from '../../../common/types/ProjectPropsContextInterface';
+import type {
+  APIError,
+  ConservationProjectConcise,
+  ConservationProjectMetadata,
+  EcosystemTypes,
+  TreeProjectConcise,
+  TreeProjectMetadata,
+} from '@planet-sdk/common';
+import type { Feature as GeoJSONFeature, Point as GeoJSONPoint } from 'geojson';
 
 import React, { useEffect, useState } from 'react';
 import DashboardView from '../../../common/Layout/DashboardView';
 import { useTranslations } from 'next-intl';
 import ProjectFilter from './components/ProjectFilter';
 import { useAnalytics } from '../../../common/Layout/AnalyticsContext';
 import { DataExplorerGridContainer } from './components/DataExplorerGridContainer';
-import { getAuthenticatedRequest } from '../../../../utils/apiRequests/api';
-import { useUserProps } from '../../../common/Layout/UserPropsContext';
 import { handleError } from '@planet-sdk/common';
 import { ErrorHandlingContext } from '../../../common/Layout/ErrorHandlingContext';
-import { useTenant } from '../../../common/Layout/TenantContext';
 import NoProjectsFound from './components/NoProjectsFound';
+import { useApi } from '../../../../hooks/useApi';
+
+type OmittedProjectProps =
+  | 'ecosystem'
+  | '_scope'
+  | 'fixedRates'
+  | 'isPublished'
+  | 'reviewScore'
+  | 'reviews'
+  | 'treeCost'
+  | 'description'
+  | 'options';
+
+type BaseProject<T> = Omit<T, OmittedProjectProps> & {
+  unit: string;
+  metadata: Record<string, unknown>;
+};
+
+type TreeProject = BaseProject<TreeProjectConcise> & {
+  unit: 'tree';
+  metadata: TreeProjectMetadata;
+};
+
+type ConservationProject = BaseProject<ConservationProjectConcise> & {
+  unit: 'm2';
+  isApproved: boolean;
+  isFeatured: boolean;
+  isTopProject: boolean;
+  countPlanted: number;
+  metadata: ConservationProjectMetadata & {
+    ecosystems: EcosystemTypes;
+  };
+};
+
+export type ProjectMapInfo<T> = GeoJSONFeature<GeoJSONPoint, T>;
+/** This evaluates to
+ * {
+ * type:"Feature",
+ * geometry:Geometry object,
+ * properties:  TreeProject | ConservationProject
+ * } */
+export type ProjectApiResponse = ProjectMapInfo<
+  TreeProject | ConservationProject
+>;
```

Comment:
1. I suggest setting this up the SDK if needed, but confirm it is needed first. Perhaps the properties omitted should be added to this endpoint as well?
2. This endpoint is also used for ManageProjects. Perhaps we should keep a common type.
3. I feel the structure is a bit complicated. How can it be simplified if it is needed?

Reply:
Will address this in a separate PR.